corehq/project_limits/migrations/0002_ratelimitedtwofactorlog.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.16 on 2020-10-13 12:47
+
+from django.db import migrations, models
+from architect.commands import partition
+
+
+def add_partitions(apps, schema_editor):
```

Comment:
@dannyroberts I also followed examples here, and am not sure why `apps` and `schema_editor` were unused params in other examples. Is it ok to remove these parameters in this case?

Reply:
My read of it from the code (not from hard knowledge I have memorized or anything) is that `migrations.RunPython` accepts as its argument a function that has two parameters named `apps` and `schema_editor` (or likely just positional arguments not referenced by name). If my read is right, then if you removed these parameters, `migrations.RunPython` would call it with the expected arguments and it would fail because the function accepts 0 arguments. So in short (again if my read is correct) it's a matter of adhering to an interface, regardless of whether these parameters are used within the function.